article.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
```

Comment:
This is not a valid PR @oneamanish. Kindly go through the contributing guidelines of how to contribute to this project.